extensions/wikia/SyntaxHighlight/SyntaxHighlightHooks.php

```diff
@@ -0,0 +1,88 @@
+<?php
+use Wikia\Util\Statistics\BernoulliTrial;
+
+class SyntaxHighlightHooks {
+
+	/**
+	 * Whether we should override GeSHi
+	 * Sampled at 1% in production
```

Comment:
This information does not belong to the description of the method. It does not say whether the sampling is applied by the method or the call of the method is somehow sampled elsewhere.  It should be placed where the sampling is actually applied, so line 14.